src/blk/CMakeLists.txt

```diff
@@ -44,6 +44,10 @@ if(WITH_ZBD)
 endif()
 
 if(WITH_BLUESTORE_PMEM)
+  if(HAVE_LIBDML)
+    target_link_libraries(blk PRIVATE dml::dml dml::dmlhl)
+  endif(HAVE_LIBDML)
```

Comment:
i'd suggest remove `HAVE_LIBDML` in the `endif()`.

Reply:
ack

Reply:
ack